temp1.js

```diff
@@ -0,0 +1,110 @@
+const {
+    Worker, isMainThread, workerData, BroadcastChannel, MessageChannel,
+  } = require('worker_threads');
+  const CoinbasePro = require('coinbase-pro');
+  const stringTable = require('string-table');
+  const moment = require('moment');
+  const log4js = require('log4js');
+  const { v4 } = require('uuid');
+  const { wsUrl } = require('./model/constants');
+  const { strategies } = require('./strategies/all_strategies');
+  const { loadConfigurationFile } = require('./utils/loadConfigurationFile');
+  const { checkEnvironmentVariables } = require('./utils/checkEnvironmentVariables');
+  const botConfiguration = loadConfigurationFile(process.argv);
+  const tickerChannel = new BroadcastChannel('ticker');
+  const candleChannel = new BroadcastChannel('candles-every-minute-past-10-minutes');
+  const logAppenders = {};
+  const appendersName = ['main'];
+  const logCategories = {};
+  strategies.forEach((strategy) => {
+    const identifier = v4();
+    logAppenders[identifier] = { type: 'file', filename: `${botConfiguration.logging.logDir}/${strategy.name().replace(' ', '-')}-${identifier.substring(0, 8)}.log` };
+    logCategories[identifier] = { appenders: [identifier], level: 'trace' };
+    appendersName.push(identifier);
+    strategy.markets().forEach((mkt) => {
+      if (globalConfig.markets.indexOf(mkt) === -1) globalConfig.markets.push(mkt);
+    });
+    strategy.logger = log4js.getLogger(identifier);
+    strategy.orderCallback = (order) => { port2.postMessage({ order }); };
+    globalConfig.strategies.push({
+      name: strategy.name(),
+      type: strategy.type(),
+      markets: strategy.markets().join(),
+      logId: identifier,
+    });
+  });
+  logAppenders.main = { type: 'file', filename: `${botConfiguration.logging.logDir}/main.log` };
+  logCategories.main = { appenders: ['main'], level: 'trace' };
+  logCategories.default = { appenders: ['main'], level: 'trace' };
+  log4js.configure({
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 0 spaces but found 2.](https://app.codacy.com/gh/amcalabretta/botbase/pullRequest?prid=7628479)
```suggestion
log4js.configure({
```